lib/apis/v2/felixconfig.go

```diff
@@ -130,9 +130,9 @@ type FelixConfigurationSpec struct {
 	// [Default: INFO]
 	LogSeveritySys string `json:"logSeveritySys,omitempty"`
 
-	IpInIpEnabled *bool `json:"ipInIpEnabled,omitempty"`
+	IPIPEnabled *bool `json:"ipInIpEnabled,omitempty" confignamev1:"IpInIpEnabled"`
 	// IpInIpMTU is the MTU to set on the tunnel device. See Configuring MTU [Default: 1440]
-	IpInIpMtu *int `json:"ipInIpMtu,omitempty"`
+	IPIPMtu *int `json:"ipInIpMtu,omitempty" confignamev1:"IpInIpMtu"`
```

Comment:
`IPIPMTU`?  urgh

Reply:
`IPIPMTU` is internally consistent - all letters in the two acronyms are capitalized. But fugly as all get-out.